components/Home.tsx

```diff
@@ -1,9 +1,91 @@
+import Link from "next/link";
 import * as React from "react";
 import { useIntl } from "react-intl";
 import { strong } from "../helpers/format";
 
 const Home: React.FunctionComponent = () => {
   const intl = useIntl();
+
+  const employeeLinks = [
+    {
+      title: intl.formatMessage({
+        defaultMessage: "GC Workplace Accessibility Passport Canada.ca page",
+        description: "Employee link text #1",
+      }),
+      link: intl.formatMessage({
+        defaultMessage:
+          "https://www.canada.ca/en/government/publicservice/wellness-inclusion-diversity-public-service/diversity-inclusion-public-service/accessibility-public-service/government-canada-workplace-accessibility-passport.html",
+        description: "Employee link #1",
+      }),
+    },
```

Comment:
Instead of creating a formatted message for the links you could do something like this:

```js
intl.formatMessage(
  {
     defaultMessage: "<link>Passport Canada.ca page</link>",
     description: "Manager link text #1",
  },
  {
     link: (msg: string) => (
       <a href={locale === "en" ? englishLink : frenchLink}></a>
     ),
  },
),
```

Reply:
> Instead of creating a formatted message for the links you could do something like this:
> 
> ```js
> intl.formatMessage(
>   {
>      defaultMessage: "<link>Passport Canada.ca page</link>",
>      description: "Manager link text #1",
>   },
>   {
>      link: (msg: string) => (
>        <a href={locale === "en" ? englishLink : frenchLink}></a>
>      ),
>   },
> ),
> ```

This is interesting and I am on the fence because technically the links are content to be managed. @JoshBeveridge @gobyrne, any opionions?